src/nlp/arrow_writer.py

```diff
@@ -362,6 +362,7 @@ def finalize(self, metrics_query_result: dict):
                 that the filter keeps only the metrics for the considered split, under the namespace `split_name`.
         """
         import apache_beam as beam
+
```

Comment:
It looks like an isort issue

Reply:
Okay, I removed them (automated tools are eager to do stuff sometimes :))